### PR TITLE
Alter on drop migrations

### PR DESCRIPTION
**Pull request by peterphan20**

### CODE REVIEW

File: src/routes/index.js [hunk 1]
```diff
@@ -1,14 +1,23 @@
-const { createUser } = require('../schemas/schemas')
+const { createUser, deleteUser } = require('../schemas/schemas')
 
 const userRoutes = async fastify => {
-  fastify.get('/users', async () => {
```
Comment (anthonybui1) [inline src/routes/index.js:4]: I believe this should be on /users

File: src/schemas/schemas.js [hunk 2]
```diff
@@ -17,4 +17,13 @@ const createUser = {
...
+const deleteUser = {
+  params: {
+    type: 'object',
+    properties: {
+      id: { type: 'string' },
+    },
+  },
+}
+
+module.exorts = { createUser, deleteUser }
```
Comment (anthonybui1) [inline src/schemas/schemas.js]: Should be module.exports

**Review by anthonybui1 — CHANGES_REQUESTED** [review]
Change these and commit again!

File: src/schemas/schemas.js (hunk 2 above)
Comment (peterphan20) [reply src/schemas/schemas.js]: Changed

**Review by anthonybui1 — APPROVED** [review]
LGTM!

**Pull request merged**